packages/asset-service/src/generateAssetData/ethTokens/uniV2.ts

```diff
@@ -0,0 +1,32 @@
+import { AssetNamespace, toAssetId, toChainId } from '@shapeshiftoss/caip'
+import { AssetDataSource, ChainTypes, NetworkTypes, TokenAsset } from '@shapeshiftoss/types'
+
+export const getUniV2Token = (): TokenAsset[] => {
+  const chain = ChainTypes.Ethereum
+  const network = NetworkTypes.MAINNET
+  const assetNamespace = AssetNamespace.ERC20
+  const assetReference = '0x470e8de2ebaef52014a47cb5e6af86884947f08c' // UNI-V2 contract address
+
+  const result: TokenAsset = {
+    assetId: toAssetId({
+      chain,
+      network,
+      assetNamespace,
+      assetReference
+    }),
+    chainId: toChainId({ chain, network }),
+    dataSource: AssetDataSource.CoinGecko,
+    name: 'Uniswap ETH/FOX LP',
```

Comment:
Similar naming comment as above. We might want product to weigh in here on desired naming. I will pose the question.

Reply:
`UNI.V2.FOX-WETH`, in-line with @reallybeard's suggestion was my favorite option, though it isn't ultimately my call.

Reply:
Trying out the fancy new product tag: @shapeshift/product, please see above.

Reply:
Posting here for additional visibility that I would prefer `UNI-V2-FOX-WETH` using dashes over dots to make it align more with typical erc20 symbol naming convention